### PR TITLE
Initial commit of app monitoring scenario.

### PR DESCRIPTION
**Pull request by adamboeglin**

### CODE REVIEW

**Comment by adamboeglin** [comment]
Acrolinx score: 90

File: docs/example-scenario/apps/app-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+1. Developers and administrators can review health, performance, and usage information.
+1. SQL Database emits telemetry.
+1. Azure Monitor collects and analyzes infrastructure metrics and quotas.
+1. Log Analytics collects and analyzes logs and metrics.
+1. Developers and administrators can review health, performance, and usage information.
+
+### Components
+
+- [Azure App Service][Azure App Service] is a PaaS service for building and hosting apps in managed virtual machines. The underlying compute infrastructures on which your apps run is managed for you. App Service provides monitoring of resource usage quotas and app metrics, logging of diagnostic information, and alerts based on metrics. Even better, you can use Application Insights to create [availability tests][availability tests] for testing your application from different regions.
+- [Application Insights][Application Insights] is an extensible Application Performance Management (APM) service for developers and supports multiple platforms. It monitors the application, detects application anomalies such as poor performance and failures, and sends telemetry to the Azure portal.
```
Comment (MikeWasson) [inline docs/example-scenario/apps/app-monitoring.md]: Should call out that App Insights is also used for:
- Application logging (exceptions, trace statements)
- [Distributed tracing and correlation](https://docs.microsoft.com/en-us/azure/application-insights/application-insights-correlation)

File: docs/example-scenario/apps/app-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+### Alternatives
+
+This article describes conveniently available monitoring options with popular features, but you have many choices, including the option to create your own logging mechanisms. A best practice is to add monitoring services as you build out tiers in a solution. Here are some possible extensions and alternatives:
+
+- Consolidate Azure Monitor and Application Insights metrics in Grafana using the [Azure Monitor Data Source For Grafana][Azure Monitor Data Source For Grafana].
+- Automate monitoring functions using [Azure Automation][Azure Automation].
+- Add communication with [ITSM solutions][ITSM solutions]).
```
Comment (MikeWasson) [inline docs/example-scenario/apps/app-monitoring.md]: Remove extra closing paren

File: docs/example-scenario/apps/app-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+Azure Monitor, Application Insights, and Log Analytics all send [alerts](/azure/monitoring-and-diagnostics/monitoring-overview-alerts). For example, Azure Monitor alerts on platform-level metrics such as CPU utilization, while Application Insights alerts on application-level metrics such as server response time. Azure Monitor alerts on new events in the Azure Activity Log, while Log Analytics can issue alerts about metrics or event data for the services configured to use it. [Alerts (Preview)](/azure/monitoring-and-diagnostics/monitoring-overview-unified-alerts) is a new, unified alerting experience in Azure that uses a different taxonomy.
+
+### Alternatives
+
+This article describes conveniently available monitoring options with popular features, but you have many choices, including the option to create your own logging mechanisms. A best practice is to add monitoring services as you build out tiers in a solution. Here are some possible extensions and alternatives:
+
+- Consolidate Azure Monitor and Application Insights metrics in Grafana using the [Azure Monitor Data Source For Grafana][Azure Monitor Data Source For Grafana].
```
Comment (MikeWasson) [inline docs/example-scenario/apps/app-monitoring.md]: There's also a [Data Dog](https://www.datadoghq.com/blog/azure-monitoring-enhancements/) connector for Azure Monitor

File: docs/example-scenario/apps/app-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+- Automate monitoring functions using [Azure Automation][Azure Automation].
+- Add communication with [ITSM solutions][ITSM solutions]).
+- Extend Log Analytics with a [management solution][management solution].
+
+### Scalability and availability
+
+This scenario focuses on PaaS solutions for monitoring in large part because they conveniently handle availability and scalability for you and are backed by service-level agreements (SLAs). For example, App Services provides a guaranteed [SLA][SLA] for its availability.
```
Comment (MikeWasson) [inline docs/example-scenario/apps/app-monitoring.md]: Should mention scaling considerations for App Insights 
- [Rate limits](https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#application-insights-limits)
- AI will throttle if you go over the limit
- Use [filtering](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-api-filtering-sampling) or [sampling](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-sampling) to reduce the data rate

**Comment by MikeWasson** [comment]
@adamboeglin  see me review comments

**Comment by adamboeglin** [comment]
Implemented changes

**Pull request merged**